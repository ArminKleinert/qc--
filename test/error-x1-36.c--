export bits1 foo;

foo() {
    bits32 x, y;
    bits16 a ,b;

    ;
}


