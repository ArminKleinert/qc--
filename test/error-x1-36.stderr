Character 0-17 Error: declared type does not match actual type
